.github/workflows/arviz_compat.yml

```diff
@@ -9,11 +9,11 @@ jobs:
   pytest:
     strategy:
       matrix:
-        python-version: ["3.8"]
```

Comment:
this had no effect, though if I do make it have an effect with
```
python-version: ${{ matrix.python-version }}
```
then I get
```
Found conflicts! Looking for incompatible packages.
```

This is a separate issue, will deal with it later